matrix_vision_bot.py.py

```diff
@@ -0,0 +1,38 @@
+import requests
+import json
+
+TOKEN = 'YOUR_TOKEN' # insert your token from BotFather here
+BASE_URL = f"https://api.telegram.org/bot{TOKEN}/"
+LONG_POLLING_TIMEOUT = 10
+
+def bot_function():
+    pass
+
+
+def main(PLACEHOLDER_MSG="Привет", any_arguments) -> None:
```

Comment:
I meant smth like this, but ok. 
```suggestion
def main():
```
We will do the rest changes in separate PR. Approved.